platforms/software/dependency-management/src/main/java/org/gradle/api/internal/artifacts/configurations/DefaultConfiguration.java

```diff
@@ -864,11 +855,11 @@ private DependencyConstraint registerConsistentResolutionConstraint(ResolvedComp
         return null;
     }
 
-    private void performPreResolveActions(ResolvableDependencies incoming) {
+    private void runBeforeResolve() {
```

Comment:
💅 I like the old name better.  Run _what_ before resolve?

Reply:
This runs a method called `beforeResolve`. I've added javadoc